packages/material-ui/src/Alert/Alert.test.js

```diff
@@ -1,15 +1,11 @@
 import * as React from 'react';
-import { getClasses, createMount, describeConformance } from 'test/utils';
+import { createMount, describeConformance } from 'test/utils';
```

Comment:
Please use the describeConformanceV5 test suite instead of this one. See how it is used in the Button.test.js